libclc/amdgcn-amdhsa/libspirv/misc/sub_group_shuffle.cl

```diff
@@ -37,7 +37,16 @@ __AMDGCN_CLC_SUBGROUP_SUB_I32(unsigned short, t);
 
 #ifdef cl_khr_fp16
 #pragma OPENCL EXTENSION cl_khr_fp16 : enable
-__AMDGCN_CLC_SUBGROUP_SUB_I32(half, DF16_);
+_CLC_DEF half _Z28__spirv_SubgroupShuffleINTELIDF16_ET_S0_j(
+    half Data, unsigned int InvocationId) {
+  union {
+    int i;
```

Comment:
This is a nit, but I'd go with `unsigned int`, even thought we're type punning back to `half`.

Reply:
ta done